index.html

```diff
@@ -18,14 +18,14 @@
 		<div class="homepage-hero__layer"></div>
 		<article class="homepage-hero__article">
```

Comment:
You can get rid off this article wrapper div. Also i recomend to rename the classes according their content. For example instead of `homepage-hero__headline` you could use `homepage-hero__hdlp-logo`.